.infrastructure/env/dev/terraform.tfvars

```
@@ -21,3 +21,10 @@ cdn_custom_headers = [
 dns_domain_name = "dev.developer.pagopa.it"
 
 use_custom_certificate = true
+
+# CMS Strapi DNS
+dns_domain_name_cms = {
+  "cms.dev.developer.pagopa.it" = {
```

```suggestion
  "cms.dev.developer.pagopa.it" = {
```
just a curiosity (I don't know if this is possible): can we use functions to set the name of the variable or it must be a string?
I mean, can we use the `dns_domain_name` variable to set the `dns_domain_name_cms` value?

Something like:
```
dns_domain_name_cms = {
  format("cms.%s", var.dns_domain_name) = {
```
in this case we are reusing something we already have.

---

I do not believe: https://github.com/hashicorp/terraform/issues/17229